test/api/unit/models/build.test.js

```diff
@@ -1,160 +1,183 @@
+    it('should add a build token', (done) => {
+      let build;
 
-      factory.site().then(site => {
+      factory.site().then((site) => {
         build = Build.build({
           site: site.id,
-        })
-        return build.validate()
+        });
+
+        return build.validate();
       }).then(() => {
-        expect(build.token).to.be.okay
-        done()
-      }).catch(done)
-    })
+        expect(build.token).to.be.okay;
+        done();
+      })
+      .catch(done);
+    });
 
-    it("should not override a build token if one exists", done => {
-      let build
+    it('should not override a build token if one exists', (done) => {
+      let build;
 
-      factory.site().then(site => {
+      factory.site().then((site) => {
         build = Build.build({
           site: site.id,
-          token: "123abc"
-        })
-        return build.validate()
+          token: '123abc',
+        });
+        return build.validate();
       }).then(() => {
-        expect(build.token).to.equal("123abc")
-        done()
-      }).catch(done)
-    })
-  })
-
-  describe("after create hook", () => {
-    it("should send a build new build message", done => {
-      const oldSendMessage = SQS.sqsClient.sendMessage
-      SQS.sqsClient.sendMessage = (params, callback) => {
-        SQS.sqsClient.sendMessage = oldSendMessage
-        done()
-      }
-      factory.build().catch(done)
-    })
-  })
-
-  describe(".completeJob(message)", () => {
-    it("should mark a build successful if there is no error", done => {
-      factory.build().then(build => {
-        return build.completeJob()
-      }).then(build => {
-        expect(build.state).to.equal("success")
-        expect(build.error).to.equal("")
-        expect(new Date() - build.completedAt).to.be.below(1000)
-        done()
-      }).catch(done)
-    })
-
-    it("should mark a build errored with a message if the error is a string", done => {
-      factory.build().then(build => {
-        return build.completeJob("this is an error")
-      }).then(build => {
-        expect(build.state).to.equal("error")
-        expect(build.error).to.equal("this is an error")
-        done()
-      }).catch(done)
-    })
-
-    it("should mark a build errored with the error's message if the error is an error object", done => {
-      factory.build().then(build => {
-        const error = new Error("this is an error")
-        return build.completeJob(error)
-      }).then(build => {
-        expect(build.state).to.equal("error")
-        expect(build.error).to.equal("this is an error")
-        done()
-      }).catch(done)
-    })
-
-    it("should update the site's publishedAt timestamp if the build is successful", done => {
-      let site
-
-      const sitePromise = factory.site()
+        expect(build.token).to.equal('123abc');
+        done();
+      })
+      .catch(done);
+    });
+  });
+
+  describe('after create hook', () => {
+    // TODO: This doesnt test anything
```

Comment:
ha, weird o_O

Looks like `SQS.sendBuildMessage` should be called with the newly created `buildId`. Want to add a quick expectation to check that?